camayoc/tests/qcs/api/__init__.py

```diff
@@ -5,10 +5,6 @@
 server. These tests should be run against QCS installations.
 
 QCS is interacted with over its API, and the base url should be specified in
-your camayoc config file with the following information:
-
-'qcs':
-    hostname: 'http://hostname_or_ip_with_port'
-
-Be sure to specify the port if you are running on a non-standard one.
+your camayoc config file. You can find a sample config file in the root of the
+repository named ``example_camayoc.yaml``.
```

Comment:
This should be updated if the example config file is renamed to `example_config.yaml`.